src/Microsoft.VisualStudio.AppDesigner/Common/Utils.vb

```diff
@@ -235,8 +235,11 @@ Namespace Microsoft.VisualStudio.Editors.AppDesCommon
             Debug.Assert(Not String.IsNullOrEmpty(throwingComponentName))
             Debug.Assert(Not String.IsNullOrEmpty(exceptionEventDescription))
 
+            ' Follow naming convention for entity name: A string to identify the entity in the feature. E.g. open-project, build-project, fix-error.
+            throwingComponentName = Regex.Replace(throwingComponentName, "([A-Z])", "-$1").TrimPrefix("-").ToLower() + "-fault"
+
             TelemetryService.DefaultSession.PostFault(
-                eventName:="vs/projectsystem/appdesigner/" & throwingComponentName.ToLower,
+                eventName:="vs/ml/cps/appdesigner/" & throwingComponentName.ToLower,
```

Comment:
These pages are for non-cps projects as well. We should probably says ml/proppages or maybe just ml/appdesigner